package.json

```diff
@@ -166,6 +166,7 @@
     "auth-header": "1.0.0",
     "aws4": "1.11.0",
     "azure-devops-node-api": "11.2.0",
+    "bsip": "0.1.9",
```

Comment:
I don't see any code changes in this PR to actually load this dependency? I was expecting to see some changed TypeScript files.

Reply:
Hi, the PR is in draft state - we're discussing the design in https://github.com/renovatebot/renovate/discussions/18872 - and as I get things I'm sure of I'm committing them and pushing. 

So there's no need to review it per se until I remove the Draft marker - though I do absolutely appreciate the input.

The bsip dependency will be used to determine the correct file name in the cargo registry dir for a given registry.